### DISCUSSION TITLE
NetBox live coding Friday, January 7th @ 6pm UTC/1pm EST

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#8249] [Announcements]
I'll be hosting another live coding session this Friday, January 7th, at 6pm UTC/1pm EST where I work on some outstanding GitHub issues and take questions from the chat. This is a great way to get exposure to the NetBox development process, or even just learn about open source development in general. Everyone is welcome to attend!

https://youtu.be/aVAPo36KPkY

You can also find recordings of past sessions [on my YouTube channel](https://www.youtube.com/channel/UCyTz3NglyCnIAhvEgat41sQ).

**Comment by bluikko** [comment]
Could not attend live due to time zone but the recorded streams are good so thank you for the effort.  
Also it was nice to see a little bit more complex change in the January session.
